<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_equipment</class>
 <widget class="QMainWindow" name="new_equipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>685</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить новое оборудование</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_repos">
        <property name="text">
         <string>Склад</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_rep"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_cell_addr">
        <property name="text">
         <string>Адрес ячейки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_cell_addr"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_name_ru">
        <property name="text">
         <string>Наименование(RU)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_name_ru"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_name_en">
        <property name="text">
         <string>Наименование(EN)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_name_en"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_manufacturer">
        <property name="text">
         <string>Производитель</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_manufacturer"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_part_num">
        <property name="text">
         <string>Артикул</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_part_num"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_equipment">
        <property name="text">
         <string>Оборудование</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_equipment"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_invent_num">
        <property name="text">
         <string>Инвентарный номер</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_invent_number"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_act_count">
        <property name="text">
         <string>Актуальное количество</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_act_count"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_min_count">
        <property name="text">
         <string>Минимальное количество</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_min_count"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_comment">
        <property name="text">
         <string>Комментарий</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_comment"/>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
